src/Subscriptions/Storage/RedisStorageManager.php

```diff
@@ -74,9 +74,13 @@ public function subscribersByTopic(string $topic): Collection
 
         return (new Collection($subscribers))
             ->map(function (?string $subscriber): ?Subscriber {
-                return is_string($subscriber)
-                    ? unserialize($subscriber)
-                    : null;
+                $deserializedSubscriber = unserialize($subscriber);
```

Comment:
Now this is potentially calling `unserialize` with `null`.

Reply:
sure i known it,
just wanted to fast 

right now it's better